app/models/bet_manager.rb

```diff
@@ -0,0 +1,4 @@
+class BetManager < ApplicationRecord
```

Comment:
I am not a fan of `BetManager` as a name. It is a little bit nebulous as to what this model is actually responsible for. 

Everyone in the office has been linked to this article at somepoint: http://objology.blogspot.ca/2011/09/one-of-best-bits-of-programming-advice.html 

Given what this Class is doing I would lean more towards calling this a:  
Prediction
Outcome
UserBet (Kind of generic like UsersBets, but slighly better since its is actually what a `User` has `Bet` 
etc